.tekton/tasks/e2e-test.yaml

```diff
@@ -31,7 +31,7 @@ spec:
       - name: APP_SUFFIX
         value: "$(params.app_suffix)"
       - name: COMPONENT_REPO_URLS
-        value: "https://github.com/redhat-appstudio-qe/devfile-sample-python-basic,https://github.com/redhat-appstudio-qe/retrodep,https://github.com/cachito-testing/pip-e2e-test,https://github.com/redhat-appstudio-qe/fbc-sample-repo,https://github.com/redhat-appstudio-qe/nodejs-no-dockerfile,https://github.com/redhat-appstudio-qe/maven-hello-world"
+        value: "https://github.com/redhat-appstudio-qe/devfile-sample-python-basic,https://github.com/redhat-appstudio-qe/retrodep,https://github.com/cachito-testing/pip-e2e-test,https://github.com/redhat-appstudio-qe/fbc-sample-repo,https://github.com/redhat-appstudio-qe/nodejs-no-dockerfile,https://github.com/redhat-appstudio-qe/maven-hello-world,https://github.com/cqi-stonesoup-test/e2e-tests-parent-image-with-both-tag-digest,https://github.com/cqi-stonesoup-test/e2e-tests-parent-image-with-digest-only,https://github.com/cqi-stonesoup-test/e2e-tests-use-latest-parent-image"
```

Comment:
@tisutisu Should all the repositories added here be under `redhat-appstudio-qe`?

Reply:
Yes, I think it would be good to them under `redhat-appstudio-qe`

Reply:
Done